src/modules/mavlink/mavlink_receiver.cpp

```diff
@@ -614,6 +618,25 @@ MavlinkReceiver::handle_message_command_int(mavlink_message_t *msg)
 }
 
 void
+MavlinkReceiver::handle_message_command_ack(mavlink_message_t *msg)
+{
+	mavlink_command_ack_t mav;
+	mavlink_msg_command_ack_decode(msg, &mav);
+
+	vehicle_command_ack_s command_ack;
+	command_ack.result = mav.result;
+	command_ack.command = mav.command;
+
+	if (_command_ack_pub == nullptr) {
+		_command_ack_pub = orb_advertise_queue(ORB_ID(vehicle_command_ack), &command_ack,
+						       vehicle_command_ack_s::ORB_QUEUE_LENGTH);
+
+	} else {
+		orb_publish(ORB_ID(vehicle_command_ack), _command_ack_pub, &command_ack);
+	}
+}
+
```

Comment:
For feedback and log purposes i think we should also register who approved the flight